modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -1237,6 +1256,11 @@ private Servlet _createServlet() throws Exception {
 			Map<Class<?>, Set<Class<?>>> classesMap =
 				processingElementsContainer.getExtensionsTypeRegistry();
 
+			processingElementsContainer.setTypeRegistry(
```

Comment:
Why here? looks like more of a field related thing that should be before _collectObjectFields